Userland/Libraries/LibC/wchar.cpp

```diff
@@ -626,4 +626,28 @@ size_t mbsrtowcs(wchar_t* dst, char const** src, size_t len, mbstate_t* ps)
     // SIZE_MAX is as close as we are going to get to "unlimited".
     return mbsnrtowcs(dst, src, SIZE_MAX, len, ps);
 }
+
+size_t wcscspn(wchar_t const* wcs, wchar_t const* reject)
+{
+    for (auto const* wc_pointer = wcs;;) {
+        auto c = *wc_pointer++;
+        wchar_t rc;
+        do {
+            if ((rc = *reject++) == c)
```

Comment:
- You're an OOB read, Harry.
- I'm a _what_?
- ��y���y���y�M��y07h6��y���y8�M��y��y���y���N��y04h_��y8HN��y���y���y���y�N��y09���y8�N��y���y���y���O��y0<h��y8HO��y���y���y���y�O��y0Mh��y8